### PR TITLE
[5.x] Update Persian translation

### PR DESCRIPTION
**Pull request by peimn**
Add missed and new translations.

### CODE REVIEW

File: resources/js/components/entries/Listing.vue [hunk 1]
```diff
@@ -218,7 +218,7 @@ export default {
 
             // When reordering, we *need* a site, since mixing them up would be awkward.
             // If we're dealing with multiple sites, it's possible the user "cleared"
-            // the site filter so we'll want to fall back to the initial site.
+            // the site filter, so we'll want to fall back to the initial site.
```
Comment (duncanmcclean) [inline resources/js/components/entries/Listing.vue:221]: This change seems unrelated - can you remove it from this PR?

File: resources/js/components/entries/PublishForm.vue [hunk 2]
```diff
@@ -558,7 +558,7 @@ export default {
 
         runAfterSaveHook(response) {
             // Once the save request has completed, we want to run the "after" hook.
-            // Devs can do what they need and we'll wait for them, but they can't cancel anything.
+            // Devs can do what they need, and we'll wait for them, but they can't cancel anything.
```
Comment (duncanmcclean) [inline resources/js/components/entries/PublishForm.vue:561]: This change seems unrelated - can you remove it from this PR?

**Review by duncanmcclean — CHANGES_REQUESTED** [review]

File: resources/js/components/entries/PublishForm.vue (hunk 2 above)
Comment (peimn) [reply resources/js/components/entries/PublishForm.vue:561]: Allright

Comment (duncanmcclean) [reply resources/js/components/entries/PublishForm.vue]: Thanks

**Review by duncanmcclean — APPROVED** [review]
